packages/plugins/documentation/server/services/helpers/build-component-schema.js

```diff
@@ -94,17 +105,26 @@ const getAllSchemasForContentType = ({ routeInfo, attributes, uniqueName }) => {
     // Build the list response schema
     schemas = {
       ...schemas,
+      [`${pascalCase(uniqueName)}ListResponseDataItem`]:{
+          type: 'object',
+          properties: {
+            id: { type: 'string' },
+            attributes: { type: 'object', properties: cleanSchemaAttributes(attributes, { addSchema, dataTypeName: `#/components/schemas/${pascalCase(uniqueName)}ListResponseDataItemLocalized`}) },
+          },
+        },
+      [`${pascalCase(uniqueName)}ListResponseDataItemLocalized`]:  {
+            type: 'object',
+            properties: {
+              id: { type: 'string' },
+              attributes: { type: 'object', properties: cleanSchemaAttributes(attributes, { addSchema }) },
+            },
+        },
```

Comment:
Are these necessary? Could you just use `ResponseDataObject` on the `ListResponse` below?

```js
[`${pascalCase(uniqueName)}ListResponse`]: {
  properties: {
    data: {
      type: 'array',
      items: {
        "$ref": `#/components/schemas/${pascalCase(uniqueName)}ResponseDataObject`
//...
```

Reply:
This is because the Type is used for the Localizations Part of the Response Object. and since we cannot Re-Use the original Object, as it would have Localizations aswell and would go in a loop kinda, i needed to move it out of there.
feel free to text me in Discord if i didn't explain it well enough 

```
 "PageListResponseDataItem": {
        "type": "object",
        "properties": {
          "id": {
            "type": "string"
          },
          "attributes": {
          ....,
          ....,
          "localizations": {
                "type": "object",
                "properties": {
                  "data": {
                    "type": "array",
                    "items": {
                      "$ref": "#/components/schemas/PageListResponseDataItemLocalized"
                    }
                  }
                }
              },
```